src/main/java/Duke.java

```diff
@@ -146,12 +160,31 @@ public static void main(String[] args) {
                     } else {
                         if (!input.contains("/at ")) {
                             printError("☹ OOPS!!! The /at of an event cannot be empty.");
-                        } else if (input.split(" ").length < 4) {
-                            printError("☹ OOPS!!! The task or /at of an event cannot be empty.");
+                        } else if (input.split(" ").length < 8) {
+                            printError("☹ OOPS!!! Follow the format: event [activity] /at [day]-[month]-[year] [hour]:[minute] to [day]-[month]-[year] [hour]:[minute]");
                         } else {
                             String description = input.replaceFirst("event ", "").split("/")[0].trim();
                             String datetime = input.replaceFirst("event ", "").split("/")[1].replaceFirst("at ", "");
-                            Task newTask = new Events(description, datetime);
+
+                            String[] datetimeSplit = datetime.split(" ");
+                            String date1 = datetimeSplit[0];
+                            String time1 = datetimeSplit[1];
+                            String date2 = datetimeSplit[3];
+                            String time2 = datetimeSplit[4];
+
+                            LocalDate dateParsed1 = LocalDate.parse(date1);
+                            LocalTime timeParsed1 = LocalTime.parse(time1);
+                            LocalDateTime datetimeParsed1 = timeParsed1.atDate(dateParsed1);
+                            LocalDate dateParsed2 = LocalDate.parse(date2);
+                            LocalTime timeParsed2 = LocalTime.parse(time2);
+                            LocalDateTime datetimeParsed2 = timeParsed2.atDate(dateParsed2);
+
+                            String newDateTime1 = datetimeParsed1.format(DateTimeFormatter.ofPattern("dd MMM yyyy HH:mma"));
```

Comment:
Probably can have meaningful variable naming instead of newDateTime1

Reply:
Yes, if not, it might be difficult to maintain your codes later.